distribution/linux/flatpak/io.github.openenroth.openenroth.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+DATAFILE="magic7.vid"
+SAVEDIR="$XDG_DATA_HOME/mm7/saves"
```

Comment:
Technically that's not saves folder. What is passed to `--user-path` is used as a base folder for everything user-generated – saves, screenshots, traces, config.

Saves go to `<user-path>/saves`.

I suggest we just set this to `$XDG_DATA_HOME/mm7`.

Reply:
done